gcmt/tests/test_gcmt.py

```diff
@@ -1,9 +1,8 @@
 import numpy as np
 import xarray
 
-from gcmt.GCMDatasetCollection import GCMDatasetCollection
-from gcmt.tests.test_gcmtools_common import *
```

Comment:
We need `all_raw_testdata`. Might have been deleted by the IDE when running optimize imports...
```from .test_gcmtools_common import all_raw_testdata```